.github/workflows/shared-release.yml

```
@@ -0,0 +1,142 @@
+name: Shared / Release
```

Optional. I still find just "release" very confusing so would prefer the less ambiguous "GitHub release" and "publish crates|npm" terms

```suggestion
name: Shared / GitHub Release
```